Terraria/NetMessage.cs

```diff
@@ -2301,6 +2301,147 @@ public static void sendWater(int x, int y)
 				}
 			}
 		}
+
+
+
+		public static void joinSyncPlayer(int i)
+		{
+			int num = 0;
+			if (Main.player[i].active)
+			{
+				num = 1;
+			}
+
+			SendData(14, -1, i, "", i, (float)num, 0f, 0f, 0, 0, 0);
+
+			SendData(4, -1, i, Main.player[i].name, i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(13, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(16, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(30, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(45, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(42, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
+			SendData(50, -1, i, "", i, 0f, 0f, 0f, 0, 0, 0);
```

Comment:
Could all of these `SendData` calls use `PacketType` enums instead of integers, for clarity?